tests/datatypes/test_str.py

```diff
@@ -609,6 +609,22 @@ def test_strip(self):
 
             print('done.')
             """)
+        
+    def test_capitalize_no_args(self):
+        self.assertCodeExecution("""
+        x = 'abcde'
```

Comment:
This is a great start, but it would be good to do a little more robust testing. At the very least, checking capital letters and whitespace are unchanged; but it would also be good to check ascii < 32 control characters, unicode characters, characters with accents, and so on. 

You don't necessarily have to have an implementation that *works* for all of those cases - but if you are adding the capitalise method, we'd like to know what cases remain to be solved. You can mark those cases with `@expectedFailure` in your test suite.